# Question: 

## How many passangers survived?

### Study Design:
#### I will be using a census study to determine how many people survived the sinking of the titanic. I would do this because I have access to all the data needed to check the WHOLE population.

## Analysis

In [7]:
import pandas as pd
titanic = pd.read_csv("titanic.csv")
titanic.head(4)

,PassengerId,Survived,Pclass,Name,Sex,Age,SibSp,Parch,Ticket,Fare,Cabin,Embarked
0,1,0,3,"Braund, Mr. Owen Harris",male,22.0,1,0,A/5 21171,7.2500,NaN,S
1,2,1,1,"Cumings, Mrs. John Bradley (Florence Briggs Th...",female,38.0,1,0,PC 17599,71.2833,C85,C
2,3,1,3,"Heikkinen, Miss. Laina",female,26.0,0,0,STON/O2. 3101282,7.9250,NaN,S
3,4,1,1,"Futrelle, Mrs. Jacques Heath (Lily May Peel)",female,35.0,1,0,113803,53.1000,C123,S


In [5]:
survived = (titanic[titanic["Survived"] == 1])
count = len(survived)
print("survived: ", count)

survived:  342


# Conclusion

### According to the data table, we can determine that the number of people that survived is 342!